src/quokka_editor_back/utils/ot.py

```diff
@@ -100,7 +91,5 @@ def apply_operation(document_content: list[str], op: OperationSchema) -> list[st
             combined = [before + middle[0] + after]
     elif op.type == OperationType.DELETE:
         combined = [before + middle[0] + after]
-    else:
-        raise Exception("Invalid Operation!!!!")
```

Comment:
Why this is removed?

Reply:
Maybe add custom exception here

Reply:
Here, an exception is not required. The Pydantic schema will validate if the operation in the body is valid